src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -0,0 +1,352 @@
+
+## Setting Up Tracing
+
+{% capture __alert_content -%}
+Sentry's Performance features are currently in beta. For more details about access to these features, feel free to reach out at [https://sentry.io/for/performance/](https://sentry.io/for/performance/).
+
+Supported SDKs for Tracing:
+- JavaScript Browser ≥ 5.13.0
+- Node.js
+- Python version ≥ 0.11.2
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+### Python
+
+To send any traces, set the `traces_sample_rate` to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+```python
+import sentry_sdk
+    
+sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
+```
+
+**Automatic Instrumentation**
+
+Many integrations for popular frameworks automatically capture traces. If you already have any of the following frameworks set up for Sentry error reporting, you will start to see traces immediately:
+
+- All WSGI-based frameworks (Django, Flask, Pyramid, Falcon, Bottle)
+- Celery
+- Redis Queue (RQ)
+
+[Spans]({%- link _documentation/performance/performance-glossary.md -%}#span) are instrumented for the following operations within a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction):
+
+- Database that uses SQLAlchemy or the Django ORM
+- HTTP requests made with `requests` or the `stdlib`
+- Spawned subprocesses
+
+If you want to enable all relevant transactions automatically, you can use this alternative configuration (currently in alpha):
+
+```python
+import sentry_sdk
+
+sentry_sdk.init("___PUBLIC_DSN___", _experiments={"auto_enabling_integrations": True})
+```
+
+**Manual Instrumentation**
+
+To manually instrument certain regions of your code, you can create a [transaction]({%- link _documentation/performance/performance-glossary.md -%}#transaction) to capture them. 
+
+The following example creates a transaction for a scope that contains an expensive operation (for example, `process_item`), and sends the result to Sentry:
+
+```python
+import sentry_sdk
+
+while True:
+  item = get_from_queue()
+
+  with sentry_sdk.start_span(op="task", transaction=item.get_transaction()):
+      # process_item may create more spans internally (see next examples)
+      process_item(item)
+```
+
+**Adding More Spans to the Transaction**
+
+The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
```

Comment:
```suggestion
The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is a currently open `transaction` and add all newly created [spans]({%- link _documentation/performance/performance-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
```